SPEC.md

```diff
@@ -2,33 +2,37 @@
 
 _For version information, see [VERSION](VERSION)_
 
-The "App Container" defines an image format, image discovery mechanism and execution environment that can exist in several independent implementations. The core goals include:
+"App Container" (appc) is a specification describing how applications can be packaged, distributed, and executed in a portable and self-contained way.
+The specification defines an image format, image discovery mechanism, and execution environment that can exist in several independent implementations.
```

Comment:
s#that can exist in several independent implementations##
This is called out as a core goal below (composable, implementation independent). Keep this sentence short and sweet?